api/nakadi-event-bus-api.yaml

```diff
@@ -1210,6 +1219,22 @@ definitions:
       - type
       - schema
 
+  EventTypeStatistics:
+    type: object
+    description: |
+      Operational statistics for an EventType. This data is generated by Nakadi based on the runtime and
+      might be used to guide changes in internal parameters.
+
+    properties:
+      expected_write_rate:
```

Comment:
@rcillo how about two fields: `time` (long) and `unit` (like Java's TimeUnit). This should cover all cases.